catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -545,4 +552,11 @@ private QueryRequestImpl getQuery(Filter filter, int index, int pageSize) {
                 TimeUnit.MINUTES.toMillis(1)), new HashMap<>());
     }
 
+    public void setStorageProvider(StorageProvider storageProvider) {
```

Comment:
Didn't know if they were still needed when using the `@Reference` annotation for injection. Will remove